### PR TITLE
(ccs/lsstcam) misc improvements

### PR DESCRIPTION
**Pull request by glennmorris**
As per [Slack](https://lsstc.slack.com/archives/C01270EHCQ3/p1689178237753639)

### CODE REVIEW

File: site/profile/manifests/ccs/data.pp [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# @summary
+#   Generic ccs /data path hierarchy
+#
+# @param files
+#   `file` resources to create.
+#
+class profile::ccs::data (
```
Comment (jhoblitt) [inline site/profile/manifests/ccs/data.pp]: This class isn't needed as we already have some special handling for a `files` key. E.g.: 

- https://github.com/lsst-it/lsst-control/blob/df28736ac804d58546938d2aa32e13daf0d54be6/hieradata/role/dnscache.yaml#L21-L28
- https://github.com/lsst-it/lsst-control/blob/df28736ac804d58546938d2aa32e13daf0d54be6/hieradata/role/rke.yaml#L21-L107

**Review by jhoblitt — CHANGES_REQUESTED** [review]
This generally looks good.  The `profile::ccs::data` class isn't needed as there is already support for file creation using the `files` key.

File: site/profile/manifests/ccs/data.pp (hunk 1 above)
Comment (glennmorris) [reply site/profile/manifests/ccs/data.pp]: Good to know, thanks. Addressed in 09bcbf98.

**Review by jhoblitt — APPROVED** [review]
Looks good.

**Comment by glennmorris** [comment]
Rebased on current production.

**Pull request merged**